docs/vue-testing-library/setup.mdx

```diff
@@ -3,4 +3,47 @@ id: setup
 title: Setup
 ---
 
-`Vue Testing Library` does not require any configuration to be used.
+`Vue Testing Library` does not require any configuration to be used. However,
+there are some things you can do when configuring your testing framework to
+reduce some boilerplate. In these docs we'll demonstrate configuring Jest, but
+you should be able to do similar things with
+[any testing framework](#using-without-jest) (Vue Testing Library does not
+require that you use Jest).
```

Comment:
Are we missing some other important setup for Jest such as `.vue` support or is that included by default?

Reply:
Hm, Single File Components requires an extra bundling step, so it might be less straightforward. For instance, we need to add "vue" to `moduleFileExtensions`, and also install vue-jest to process *.vue files. 

Docs from the official library: https://vue-test-utils.vuejs.org/installation

Reply:
I just had to set this up today and I can confirm that for single file components it does require following the steps described on the link shared by @afontcu above.